core/src/main/scala/org/apache/spark/sql/execution/ObjectHashMapAccessor.scala

```diff
@@ -1311,10 +1309,9 @@ case class ObjectHashMapAccessor(@transient session: SnappySession,
     case StringType if !multiMap =>
       // copy just reference of the object if underlying byte[] is immutable
       val stringVar = resultVar.value
-      val platformClass = classOf[Platform].getName
       val bytes = ctx.freshName("stringBytes")
       s"""byte[] $bytes;
-        if ($stringVar.getBaseOffset() == $platformClass.BYTE_ARRAY_OFFSET
+        if ($stringVar.getBaseOffset() == Platform.BYTE_ARRAY_OFFSET
```

Comment:
Would not it cause problem  ? Actual byte[] will be accessed in different JVM and driver is creating the code. There will be a high chance of mismatch as drivers will be generally will be less than 32G.